andela/vale-ah-frontend
# #165182886 Add All Recipes Feature

#### What does this PR do?
Adds all recipes feature
#### Description of Task to be completed?
- read/access recipes created by other cooks
#### How should this be manually tested?
- ensure that all tests pass on `Travis CI`
- check the [review app](https://vale-ah-frontend-staging-pr-15.herokuapp.com) and make sure that all is well
#### Any background context you want to provide?
Recipes form the backbone of our application and introduce us to CRUD which is the building block of most web applications.
#### What are the relevant pivotal tracker stories?
[#165182886](https://www.pivotaltracker.com/story/show/165182886)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
